apps/examples/alc5658char_test/Makefile

```diff
@@ -60,7 +60,7 @@ include $(APPDIR)/Make.defs
 # built-in application info
 
 APPNAME = alctest
-REALNAME = alc5658char_test
+FUNCNAME = alc5658char_test
```

Comment:
FUNCNAME should be a real main function name so that alc5658char_test_main is right.